app/js/skrollr.js

```diff
@@ -29,17 +29,30 @@ angular.module("sn.skrollr", [])
          */
         function($window, $document, $rootScope) {
 
-            return {
+            var s,
+                serviceMethods,
+                init = 0;
+
+            serviceMethods = {
                 init: function() {
 
                     $document.ready(function () {
                         $rootScope.$apply(function() {
```

Comment:
Also a good idea to comment your code to explain your functions and variables, their purpose and your thinking. Even if it seems simple, you can't assume it will be obvious to everyone who looks at your code.